src/main/java/net/countercraft/movecraft/combat/features/tracking/events/CraftSunkByEvent.java

```diff
@@ -56,7 +61,17 @@ public String causesToString() {
         StringBuilder stringBuilder = new StringBuilder();
         stringBuilder.append(((PlayerCraft) this.craft).getPilot().getDisplayName());
         stringBuilder.append(" ").append(I18nSupport.getInternationalisedString("Killfeed - Sunk By")).append(" ");
-        stringBuilder.append(NameUtils.offlineToName(latestDamage.getCause()));
+        Player cause = (Player) latestDamage.getCause();
+        boolean checkPerm = ((PlayerCraft) this.craft).getPilot().hasPermission("movecraft.combat.bypassobfuscation");
```

Comment:
No need for `this.craft`, we can just use `craft`.  The only time we need to specify `this._` is when there is a conflict for `_`.

Reply:
Also, is it intended for this permission to bypass the ability to obfuscate your own death messages?  I'd prefer this be setup as an admin override that removes all obfuscation.